frontend/src/app/components/users/birthday/users.birthday.controller.js

```diff
@@ -1,29 +1,40 @@
 
 import config from '../../../../config/environment';
 
-const defaultImg = require('../../../../resources/default.jpg');
-
+/**
+ * Controller for Birthday
+ */
 export default class BirthdayController {
     constructor($interval, $http, $routeParams) {
         'ngInject';
         this.$interval = $interval;
         this.$http = $http;
         this.$routeParams = $routeParams;
         this.employee_id = this.$routeParams.id;
+        this.employee_firstName = this.$routeParams.firstName;
+        this.employee_lastName = this.$routeParams.lastName;
+        this.employee_photoUrl = this.$routeParams.photoUrl;
         this.employees = [];
         this.baseUri = `http://${config.serverHost}:${config.serverPort}`;
         this.employeeRoute = `${this.baseUri}/crud/employees`;
         if (this.employee_id === undefined || this.employee_id === null) {
-            this.urlBirthday = `${this.employeeRoute}/birthdaylist`;
+            if (this.employee_firstName === undefined && this.employee_lastName === undefined && this.employee_photoUrl  === undefined) {
```

Comment:
Use falsy conditional or comparison
Not 'if (varible === undefined)'
Instead 'if (!variable)'

Reply:
I fixed your observations, please could you review again?